src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Ricci.hpp

```diff
@@ -0,0 +1,61 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///\file
+/// Declares function templates to calculate the Ricci tensor
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace GeneralizedHarmonic {
+
+// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Compute spatial Ricci tensor using evolved variables and
+ * their first derivatives.
+ *
+ * \details Compute \f$R_{ij}\f$ as:
+ *
+ * \f{align}
+ * R_{i j} =& \case{1}{2} g^{a b} \left( \partial_{(i} d_{a b j)}
```

Comment:
why not use \frac{1}{2}? I'm not sure what \case{1}{2} does here

Reply:
thats an autocorrect error, it should have been `\frac` anyway. Fixed it.